galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -353,3 +353,15 @@ messages:
     level: Warning
     description: "Virtual IP addresses are required for ports serving TCP (or unset) protocol"
     template: "ServiceEntry addresses are required for this protocol."
+
+  - name: "IngressRouteRulesNotAffected"
+    code: IST0135
+    level: Warning
+    description: "Virtualservice not affected Ingress route"
```

Comment:
More information is needed in the description.  I have not faced this problem and am not certain what the text should be here.  Is there a page of Istio documentation explaining the problem?  We need to use similar wording so that a user seeing a help page for this message will understand this misconfiguration.

Reply:
Thanks.  Perhaps the description could be exactly what is in the doc, "Route rules have no effect on ingress gateway requests".